docs/windows.md

```diff
@@ -426,6 +361,128 @@ antrea-agent-windows-6hvkw                             1/1     Running     0
 kube-proxy-windows-2d45w                               1/1     Running     0          102s
 ```
 
+### Add Windows kube-proxy DaemonSet (only for Kubernetes versions prior to 1.26)
+
+Starting from Kubernetes 1.26, Antrea will no longer support Windows kube-proxy due to its lack of support for the kernel datapath, which Antrea requires.
+You should skip this step on the kubernetes cluster starting from 1.26 and enable proxy-all mode for windows agent.
+
+Add Windows-compatible versions of kube-proxy by applying file `kube-proxy.yaml`.
+
+Download `kube-proxy.yaml` from kubernetes official repository and set
+kube-proxy version.
+
+```bash
+# Example:
+curl -L "https://github.com/kubernetes-sigs/sig-windows-tools/releases/download/v0.1.5/kube-proxy.yml" | sed 's/VERSION-nanoserver/v1.20.0/g' > kube-proxy.yml
+```
+
+Replace the content of `run-script.ps1` in configmap named `kube-proxy-windows`
```

Comment:
Can we automate this part with a Powershell script, and even auto-download kube-proxy.yml? It looks complicated.

@wenyingd

Reply:
Since we don't plan to support kube-proxy userspace mode any more, maybe it is not worth to add automation for changes with kube-proxy on Windows?

Reply:
Sure, we can keep it manual then.